include/mxnet/ndarray.h

```diff
@@ -986,9 +986,8 @@ class NDArray {
 #endif
         delay_alloc = false;
       } else if (shandle.size < dbytes) {
-        // free storage if necessary and alloc again
-        if (shandle.size > 0) Storage::Get()->Free(shandle);
-        // init storage
```

Comment:
nit: deleted previous comment for initializing storage after freeing it.

Reply:
I do not quite get what you are suggesting. I merged the previous comments into one line. Basically we free first and alloc again, which is a resize operation.

Reply:
Ohh ... sorry if I wasn't clear. I meant that in the previous version of code there waas this comment line:
// init aux storage
before
shandle = Storage::Get()->Alloc(dbytes, shandle.ctx);
 which got deleted in your commit. Suggesting that it would be good idea to add it back. Earlier comment already mentioned "// free storage if necessary and alloc again" and still used "// init storage". Anyways its not that big a deal. Current comment still looks reasonable.

Reply:
I see. The original one was not accurate. "free storage if necessary and alloc again" was not true for the Free function. I will leave as it is for now.